extensions/amp-story/1.0/amp-story-shadow-reset.css

```diff
@@ -17,4 +17,5 @@
 :host {
   all: initial !important;
   color: initial !important;
+  visibility: inherit !important;
```

Comment:
Do we actually want to do `inherit` in the shadow reset?  Wouldn't we want to `initial` to be the default behavior, but override this in the consent shadow DOM?

Reply:
PTAL